omod/src/main/webapp/resources/js/view/bill.js

```diff
@@ -302,7 +302,11 @@ define(
 					total: total,
 					totalPaid: totalPaid,
 					formatPrice: openhmis.ItemPrice.prototype.format,
-					__: i18n }));
+					__: i18n
+				}));
+				if (total > 0) {
```

Comment:
Does the page always get refreshed if the total is zero? Is there any case where this will leave the amount filled with the wrong remaining amount due?